### ISSUE TITLE
$whatif queries no longer work in Graph Explorer

### ISSUE THREAD

**Issue by MIchaelMainer** [microsoftgraph/microsoft-graph-explorer-v4#744]
**Describe the bug**
Graph Explorer expects that all query parameters are submitted with a value. This breaks the use of $whatif and other query parameters that don't use a value.

**To Reproduce**
1. Go to Graph Explorer ad submit the following request: `https://graph.microsoft.com/v1.0/users?$whatif`

**Expected behavior**
The response looks like:

```json
{
    "Description": "Execute HTTP request",
    "Uri": "https://graph.windows.net/v2/fe639da9-7429-46bf-9a68-40cfc8229591/users?$select=businessPhones,displayName,givenName,jobTitle,mail,mobilePhone,officeLocation,preferredLanguage,surname,userPrincipalName,id",
    "HttpMethod": "GET",
    "TargetWorkloadId": "Microsoft.DirectoryServices"
}
```

**Actual behavior**

Graph Explorer submits `GET https://graph.microsoft.com/v1.0/users?$whatif=undefined` which results in the service API ignoring the $whatif parameter. This means I can't discover the underlying workload. The service processes the request as if it was done without the whatif parameter.

**Comment by thewahome** [comment]
@MIchaelMainer this works in the staging environment. It was fixed in this [PR](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/708) . It should move to production in the next release.

**Comment by thewahome** [comment]
The changes are in production and this issue fixed by [release v4.1.10](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/releases/tag/v4.1.10)
